### ISSUE TITLE
Pressing spacebar makes playback speed 2x instead of play/pause

### ISSUE THREAD

**Issue by UstedLoco** [uBlockOrigin/uBlock-issues#2984]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is NOT an issue with [YouTube](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [Facebook](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [Twitch](https://www.github.com/uBlockOrigin/uAssets/issues/5184).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] using a new, unmodified browser profile.

### Description

Pressing spacebar makes playback speed 2x instead of play/pause

### A specific URL where the issue occurs.

youtube.com

### Steps to Reproduce

open a youtube video, press spacebar, video plays at double speed instead of pausing

### Expected behavior

pressing spacebar pauses and resumes playback

### Actual behavior

pressing spacebar results in video playing in double speed

### Configuration

<details>

```yaml
uBlock Origin: 1.53.0
Chromium: 119
filterset (summary):
 network: 133520
 cosmetic: 216996
 scriptlet: 46448
 html: 0
listset (total-discarded, last-updated):
 added:
  https://filters.adtidy.org/extension/ublock/filters/14.txt: 75393-117, 2d.15h.36m
  adguard-generic: 80331-495, 2d.15h.40m
  adguard-mobile: 8762-50, 2d.15h.39m
  adguard-social: 22031-930, 2d.15h.38m
  adguard-spyware-url: 1337-5, 2d.15h.37m
  block-lan: 55-0, 1d.21h.33m
  fanboy-cookiemonster: 48997-3805, 2d.19h.44m
  fanboy-thirdparty_social: 69-3, 3d.19h.49m
  ublock-annoyances: 5892-231, 2d.19h.43m
 default:
  user-filters: 16-0, never
  easylist: 75532-1740, 4d.17h.3m
  easyprivacy: 32964-237, 13h.27m
  plowe-0: 3718-1167, 4d.21h.5m
  ublock-badware: 7494-321, 13h.26m
  ublock-filters: 35752-4231, 13h.25m
  ublock-privacy: 739-93, 2d.19h.42m
  ublock-quick-fixes: 175-52, 5h.22m
  ublock-unbreak: 2154-97, 13h.24m
  urlhaus-1: 9445-0, 9h.23m
filterset (user): [array of 16 redacted]
trustedset:
 added: [array of 117 redacted]
 removed:
  chrome-extension-scheme
  edge-scheme
switchRuleset:
 added: [array of 3 redacted]
userSettings: [none]
hiddenSettings: [none]
supportStats:
 allReadyAfter: 583 ms (selfie)
 maxAssetCacheWait: 366 ms
```
</details>

**Comment by iam-py-test** [comment]
Have you tried disabling uBo and seeing if the problem persists?
Thanks

**Comment by gorhill** [comment]
uBO doesn't install event handler for keyboard on websites.

Please investigate more on your side, something else is causing this.

**Comment by UstedLoco** [comment]
Yeah, disabled uBo, issue disappears.

**Comment by iam-py-test** [comment]
You have several custom filterlists. Please reset to default settings.
Thanks

**Comment by UstedLoco** [comment]
Ok thanks, will investigate further.

**Comment by krystian3w** [comment]
Maybe this is new small A/B test to disable wide spectrum blockers (instad of pop-up/prompts and fake lags/delay).

**Comment by gwarser** [comment]
This is a new YT feature. Nothing to do with uBO. Short "click" pauses the video - holding the button speeds it up. Maybe clean your keyboard (sticky key)?

**Comment by gwarser** [comment]
I did not. It's experiment - enabled randomly. Try adding to "My filters" `www.youtube.com##+js(set, yt.config_.EXPERIMENT_FLAGS.service_worker_enabled, false)` - it will be turned off.

**Comment by gorhill** [comment]
@gwarser You're right, I didn't know such feature existed in Youtube, holding the space bar double playback speed. Of course this also happens without uBO installed.

**Comment by gwarser** [comment]
There is even an indicator on top of the player:

![image](https://github.com/uBlockOrigin/uBlock-issues/assets/886325/725323dc-d5e0-451d-80c1-57e43cdaca88)

**Comment by UstedLoco** [comment]
I did test with all custom filters disabled and the issue persisted, but I'll experiment more.
--

**Comment by krystian3w** [comment]
This can only be helped by logging out of your youtube account - assuming that the experiment has been activated for your account and not in cookies/localStorage which you could clean once a day if you do not want to add yourself:

```adb
www.youtube.com##+js(set, yt.config_.EXPERIMENT_FLAGS.web_speedmaster_spacebar_control, false)
```
